codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithypython/localservice/customize/ErrorsFileWriter.java

```diff
@@ -206,6 +206,69 @@ def __eq__(self, other: Any) -> bool:
                         getattr(self, a) == getattr(other, a)
                         for a in attributes
                     )
+
+            class OpaqueWithTextError(ApiError[Literal["OpaqueWithTextError"]]):
+                code: Literal["OpaqueWithTextError"] = "OpaqueWithTextError"
+                obj: Any  # As an OpaqueWithTextError, type of obj is unknown
+
+                def __init__(
+                    self,
+                    *,
+                    obj,
+                    objMessage
```

Comment:
```suggestion
                    obj_message
```

Reply:
Ditto for other native types in this class; let's prefer snake-case for Smithy-generated code